code/game/machinery/bots/secbot.dm

```diff
@@ -724,6 +724,11 @@ Auto Patrol: []"},
 /obj/machinery/bot/secbot/proc/assess_perp(mob/living/carbon/human/perp as mob)
 	var/threatcount = 0 //If threat >= 4 at the end, they get arrested
 
+	if(perp.alpha <= 1) //perp is invisible, hence innocent
+		return 0
+	if(see_invisible < perp.invisibility) // for future compatibility
```

Comment:
Nevermind what I said, `see_invisible` is only for mobs. Remove this but do keep the alpha 1 thing.